.github/workflows/release.yml

```diff
@@ -453,7 +453,8 @@ jobs:
         modules: qtcharts
     - name: Configure and compile MNE-CPP
       run: |
-        ./tools/building/build_project.bat
+        cmake -DCMAKE_BUILD_TYPE="Release" -DWITH_CODE_COV=ON -B build -S src 
```

Comment:
Release mode and coverage on?
Will --coverage override build mode and make it Debug?